bot/tests/test_remote.py

```diff
@@ -351,7 +351,6 @@ def test_unsupported_analyzer(mock_config, mock_revision, mock_workflow, mock_ba
     )
     issues = mock_workflow.run(mock_revision)
     assert len(issues) == 0
-    assert mock_revision._state == BuildState.Fail
```

Comment:
The assertion should change to something like BuildState.Pass if we now expect this to be an acceptable behavior